logging/config/efk-logging-curator.adoc

```diff
@@ -5,109 +5,18 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-The Elasticsearch Curator tool performs scheduled maintenance operations on a per-project basis. Curator performs actions daily based on its configuration. Only one Curator Pod is
-recommended per Elasticsearch cluster.
+The Elasticsearch Curator tool performs scheduled maintenance operations
+on a per-project basis. Curator performs actions daily based on its configuration.
+Only one Curator Pod is recommended per Elasticsearch cluster.
```

Comment:
Does Curator perform operations *daily*? User can set the schedule to run weekly, monthly, etc.

Reply:
@mburke5678 This is a really good point. In ansible we only exposed `hour` and `minute`. Not the rest.

@ewolinetz @jcantrill CR change for 4.2 to be more in line with 3.x?
For now I'd say we document that users should run curator daily, therefore only modify run_hour and run_minute.

Reply:
i'm not sure if that's what we'd want to do. is there really a reason we would let users configure how often curator runs vs what we have as our default now? (which is to run multiple times a day)